app/code/Magento/Catalog/view/adminhtml/templates/catalog/product/edit/action/inventory.phtml

```diff
@@ -132,7 +132,7 @@
                     <div class="field">
                         <input type="text" class="input-text validate-number" id="inventory_min_sale_qty"
                                name="<?= /* @escapeNotVerified */ $block->getFieldSuffix() ?>[min_sale_qty]"
-                               value="<?= /* @escapeNotVerified */ $block->getDefaultConfigValue('min_sale_qty') * 1 ?>"
+                               value="<?= /* @escapeNotVerified */ (int) $block->getDefaultConfigValue('min_sale_qty') * 1 ?>"
```

Comment:
I don't get this. How original implementation produces a non-numeric value?

Reply:
the store config allows you to set this value at the customer group level, something that is not supported at the individual product level. something that is visible when editing the inventory config at the product level.
**Product inventory using store config**
<img width="846" alt="screenshot 2018-11-07 at 01 28 11" src="https://user-images.githubusercontent.com/12674247/48114355-bc46e000-e22c-11e8-9788-9b2f012d272f.png">
  
**Product inventory after un-checking use default**
<img width="841" alt="screenshot 2018-11-07 at 01 28 26" src="https://user-images.githubusercontent.com/12674247/48114402-e26c8000-e22c-11e8-8e1e-952d01890d0b.png">

The error occurs because from multiplying json, from the store config, by 1.

Reply:
So after the fix some JSON string is casted to `0`?

Reply:
yes

Reply:
@pmclain yes, I think so. And also please preserve old logic instead of casting:
`is_numeric(...) ? getConfig() * 1 : <DEFAULT GLOBAL VALUE>`